devices/tuya.js

```diff
@@ -354,7 +355,9 @@ module.exports = [
         model: 'TS0202',
         vendor: 'TuYa',
         description: 'Motion sensor',
-        whiteLabel: [{vendor: 'Mercator Ikuü', model: 'SMA02P'}, {vendor: 'TuYa', model: 'TY-ZPR06'}],
+        whiteLabel: [{vendor: 'Mercator Ikuü', model: 'SMA02P'},
+            {vendor: 'TuYa', model: 'TY-ZPR06'},
+            {vendor: 'Tesla Smart', model: 'Tesla Smart Sensor Motion'}],
```

Comment:
For the `model`, please fill in the model number that is listed on the device.

Reply:
Label on the device reads TS0202, so I updated it to that